cosmos-sdk-proto/src/type_urls.rs

```diff
@@ -140,3 +140,7 @@ impl TypeUrl for cosmwasm::wasm::v1::MsgUpdateAdminResponse {
 impl TypeUrl for cosmwasm::wasm::v1::MsgClearAdminResponse {
     const TYPE_URL: &'static str = "/cosmwasm.wasm.v1.MsgClearAdminResponse";
 }
+
+impl TypeUrl for ibc::applications::transfer::v1::MsgTransfer {
+    const TYPE_URL: &'static str = "ibc.applications.transfer.v1.MsgTransfer";
```

Comment:
Needs a leading slash:

```suggestion
    const TYPE_URL: &'static str = "/ibc.applications.transfer.v1.MsgTransfer";
```

Reply:
Thanks.